### PR TITLE
Fix: auto-ack flag always set true

### PR DESCRIPTION
**Pull request by rdhabalia**
### Motivation

cli should have primitive boolean for any flag but it wasn't there for auto-ack because of that we were also setting auto-ack flag true. because of below logic

### CODE REVIEW

**Comment by rdhabalia** [comment]
> The default value of auto ack should be true.

right now, there is no way, we can set this flag false because it's a flag and it can't accept any value. and in cli default value will be false. So, it seems we might need to rename it to `disable-auto-ack` and default value of this flag will be false which makes auto-ack true.

**Comment by srkukarni** [comment]
+1 to the approach

File: pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java [hunk 1]
```diff
@@ -312,7 +312,7 @@ void processArguments() throws Exception {
         @Parameter(names = "--autoAck", description = "Whether or not the framework will automatically acknowleges messages", hidden = true)
         protected Boolean DEPRECATED_autoAck = null;
         @Parameter(names = "--auto-ack", description = "Whether or not the framework will automatically acknowleges messages")
-        protected Boolean autoAck ;
+        protected boolean autoAck ;
```
Comment (merlimat) [inline pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java]: @rdhabalia With JCommander shouldn't it be able to handle boolean flags with default value to `true` ?

Eg. with : 

```java
protected boolean autoAck = true;
```

shouldn't be working to pass `--auto-ack=false`  ?

Comment (merlimat) [reply pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java]: http://jcommander.org/#_boolean

**Comment by rdhabalia** [comment]
> shouldn't be working to pass --auto-ack=false ?

uumm.. I get `Unknown option: --auto-ack=false` .. I don't think it's valid command. you can also quickly verify in [CmdFunctionsTest](https://github.com/apache/incubator-pulsar/blob/master/pulsar-client-tools-test/src/test/java/org/apache/pulsar/admin/cli/CmdFunctionsTest.java#L197) by adding `--auto-ack=false` and I can see the same error.

**Comment by rdhabalia** [comment]
@merlimat @srkukarni fixed it.

**Comment by rdhabalia** [comment]
rerun tests

**Pull request merged**